upgrade/breaking-changes/4-0-0 .md

```diff
@@ -0,0 +1,308 @@
+---
+title: 4.0.0
+description: Handle the changes in the43.0.0 release of the Telerik UI for Blazor components.
+page_title: Breaking Changes in 4.0.0
+slug: changes-in-4-0-0
+position: 1
+---
+
+# Breaking Changes in 4.0.0
+
+## Common Changes
```

Comment:
Currently there is a discrepancy in our approach with regard to identical changes in multiple components. The `Stretched` value is in the Common section. On the other hand, the `Shape` parameter is repeated in each component section.

Reply:
Agreed, moved the Shape parameter to the common changes.